docs/CONFIGURING_GLOBAL_HPKE_KEYS.md

```diff
@@ -0,0 +1,98 @@
+# Global HPKE Keypairs
+Janus has support for HPKE key advertisement on `/hpke_configs` that are not
+tied to tasks. This is mainly to support the Taskprov extension, but we are
+also [considering having this replace per-task keys entirely][1].
+
+If a global key is configured, requests to `/hpke_configs` that don't provide
+the task ID will provide the list of active global HPKE keys.
+
+This document describes the operational overhead of managing these keys.
+
+[1]: https://github.com/divviup/janus/issues/1641
+
+## Lifecycle
+A global keypair has three states:
+- `pending`: The key is in the database, but is not being advertised to clients.
+- `active`: The key is used by clients, and clients should use it to encrypt
+  reports.
+- `expired`: The key is not advertised to client and will eventually be deleted.
+
+These states are to facilitate key caching and rotation. The lifecycle of a key
+is as follows:
+1. The key is created in the `pending` state.
+1. The in-memory caches of all Janus replicas must pick up the new key. This can
+   be done by waiting or restarting the replicas.
```

Comment:
Back in ENPA, we solved a similar problem by making worker processes exit hourly so they'd restart and pick up fresh configuration values from their environment. I think we've discussed doing something similar for Janus. Can you find a relevant issue if it exists and link it here? If no such issue exists, write one up.